docs/source/gbd2017_models/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -268,7 +270,16 @@ Model Scope
 The scope of the Vivarium modeling strategy detailed in this document is to 
 sample the hemoglobin concentration for an individual simulant (who is not a 
 woman of reproductive age) and evaluate if that simulant's hemoglobin 
-concentration will respond to iron supplementation.
+concentration will respond to iron supplementation (iron responsive). 
+
+The modeling strategy detailed in this document aims to evaluate *all* iron 
+responsive anemias rather than dietary iron deficiency anemia, specifically. 
+However, the modeling strategy can be modified to include only dietary iron 
+deficiency (PAF of 1 cause), if desired by assuming that dietary iron 
+deficiency anemia is the only iron responsive cause of anemia, rather than all 
+of the causes listed in the `iron responsive anemia causes table`_ and that the 
+dietary iron deficiency anemia sequelae are the only iron responsive anemia 
+sequelae in the `anemia sequelae IDs table`_.
```

Comment:
I'm assuming i'll understand this once i've read to the end of this section, or if i read this top to bottom. but note that this is confusing---

You write both that:
- the modeling strategy in this doc aims to evaluate all iron responsive anemias
- the vivarium modeling strategy is is for dietary iron deficiency

does that mean, either, that
- you _have_ assumed_, in the way you describe above, that that dietary iron deficiency anemia is the only iron responsive cause of anemia? or,
- when you say 'modeling strategy detailed in this document' on line 275, you mean the entire document, not the vivarium modeling strategy?

Reply:
Ah, I think this is me being lazy with my wording. This document is a modeling strategy to model all iron responsive anemia rather than the singular dietary iron deficiency anemia cause; however, it can be restricted so that it only includes the singular dietary iron deficiency anemia cause by ignoring the other iron responsive anemia causes. I will clarify this in the document, but let me know if this still doesn't make sense.

Reply:
I just pushed a new commit to resolve the other concerns, but I wasn't sure how to address this comment, so if necessary, this part could still be reworded for clarity.

Reply:
oh oops! i never looked back at this, apologies.

Yes! The wording is now clear to me. Looks good @NathanielBlairStahn @alibow